### PR TITLE
Update inventory to patternfly-v4

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

File: packages/inventory/src/Inventory.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ import { Provider } from 'react-redux';
 import TagWithDialog from './TagWithDialog';
 export const InventoryContext = createContext('inventory');
 
+console.log(React, 'Inventory version!');
```
Comment (rvsia) [inline packages/inventory/src/Inventory.js:15]: Console.log

Comment (karelhala) [reply packages/inventory/src/Inventory.js]: Nice catch!

File: packages/inventory/src/InventoryList.scss [hunk 2]
```diff
@@ -8,11 +8,19 @@
...
-        .pf-c-data-toolbar__item:last-child { @include rem('min-width', 200px); }
+
+.ins-c-inventory__table--toolbar {
+    &.ins-m-footer {
+        margin: 0;
+        padding: 0;
+    }
+
+    &:not(.ins-c-inventory__table--toolbar-has-items) {
+        .pf-c-data-toolbar__content {
```
Comment (Hyperkid123) [inline packages/inventory/src/InventoryList.scss:19]: The data toolbar component no longer exists. These classNames should change to the toolbar.

Comment (karelhala) [reply packages/inventory/src/InventoryList.scss:19]: Huh, I thought I already fixed them...

File: packages/remediations/config/webpack.constants.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 module.exports = {
     externals: [
         /^@patternfly\/.*/,
-        /^@redhat-cloud-services\/.*/,
```
Comment (Hyperkid123) [inline packages/remediations/config/webpack.constants.js:4]: Is this intentional?

**Pull request merged**